src/helpers/App.php

```diff
@@ -96,15 +106,15 @@ public static function parseEnv(string $str = null)
         }
 
         if (preg_match('/^\$(\w+)$/', $str, $matches)) {
-            $value = App::env($matches[1]);
-            if ($value !== false) {
-                switch (strtolower($value)) {
+            $str = static::env($matches[1]);
+
+            if (is_string($str)) {
+                switch (strtolower($str)) {
                     case 'true':
                         return true;
                     case 'false':
                         return false;
                 }
-                $str = $value;
             }
```

Comment:
Changed this check, as `App::env` will now return `null` when it can't find a var, not `false`.